docs/example-scenario/hpc/video-rendering.md

```diff
@@ -0,0 +1,187 @@
+
+### Availability
+Monitoring of the Azure Batch components is available through a range of services, tools and APIs. This is discussed further in the [Monitor Batch solutions][batch-monitor] article.
+
+
+### Scalability
+
+Pools within an Azure Batch account can either scale through manual intervention or, by using an formula based on Azure Batch metrics, be scaled automatically. For more information on this, please see the article
+[Create an automatic scaling formula for scaling nodes in a Batch pool][batch-scaling].
+
+### Security
+
+For a deeper discussion on [security][security] please see the relevant article in the architecure center.
+
+### Resiliency
+
+While there is currently no failover capability in Azure Batch, we recommend using the following steps to ensure availbility in case of an unplanned outage:
+
+* Create a Azure Batch account in an alternate Azure location with an alternate Storage Account
+* Create the same node pools with the same name, with 0 nodes allocated
+* Ensure Applications are created and updated to the alternate Storage Account
+* Upload input files and submit jobs to the alternate Azure Batch account
+
+## Pricing
+
+Explore the cost of running this solution, all of the services are pre-configured in the cost calculator.  To see how the pricing would change for your particular use case change the appropriate variables to match your expected needs.
+
+The cost of using Azure Batch will depend on the VM sizes that are used for the pools and how long these are allocated and running, there is no cost associated with an Azure Batch account creation. Storage and data egress should also be taken into account as these will apply additional costs. The following are examples of costs that could be incurred for a job that completes in 8 hours using a different number of servers:
+
+
+- 100 High Performance CPU VMs: [Cost Estimate][hpc-est-high]
+
+  100 x H16m (16 cores, 225GB RAM, Premium Storage 512GB), 2 TB Blob Storage, 1 TB egress
+
+- 50 High Performance CPU VMs: [Cost Estimate][hpc-est-med]
+
+  50 x H16m (16 cores, 225GB RAM, Premium Storage 512GB), 2 TB Blob Storage, 1 TB egress
+
+- 10 High Performance CPU VMs: [Cost Estimate][hpc-est-low]
+  
+  10 x H16m (16 cores, 225GB RAM, Premium Storage 512GB), 2 TB Blob Storage, 1 TB egress
+
+### Low Priority VM Pricing
+
+Azure Batch also supports the use of Low Priority VMs* in the node pools, which can potentially provide a substantial cost saving. For a price comparison between standard VMs and Low Priority VMs, and to find out more about Low Priority VMs, please see [Batch Pricing][batch-pricing].
+
+\* Please note that only certain applications and workloads will be suitable to run on Low Priority VMs.
+
+## Related Resources
+
+[Azure Batch Overview][batch-overview]
+
+[Azure Batch Documentation][batch-doc]
+
+[Using containers on Azure Batch][batch-containers]
+
+Other resources that are relevant that aren't linked from else where in the doc.
```

Comment:
There is nothing below this, either remove it or move it to the top of the section